test/e2e/cli/cli_test.go

```diff
@@ -165,6 +176,131 @@ var _ = Describe("cli operation", func() {
 			Expect(out).To(ContainSubstring("e2e-test-fleet"))
 		})
 	})
+
+	Context("Resources lifecycle for", func() {
+		It("Device, Fleet, ResourceSync, Repository, EnrollmentRequest, CertificateSigningRequest", Label("75506"), func() {
+			By("Verify there are no resources created")
+			err := harness.CleanUpAllResources()
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Device:Resources lifecycle")
+			By("Apply, edit, delete device")
+			out, err := harness.CLI("apply", "-f", util.GetExamplesYamlPath("device.yaml"))
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(out).To(MatchRegexp(`(200 OK|201 Created)`))
```

Comment:
Little detail, maybe this out should be moved one line up since it is part of the previous code block

Reply:
done